common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +632,22 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
```

Comment:
Please remove this comment as I believe it is the result of copy/paste and there is no usage of tag or direct usage of `enterprise_learner` here